codes/quantum/qubits/translationally_invariant_stabilizer.yml

```diff
@@ -0,0 +1,53 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: translationally_invariant_stabilizer
+physical: qubits
+logical: qubits
+
+name: 'Translationally invariant stabilizer code'
+introduced: '\cite{arXiv:1607.01387}'
+
+description: |
+  A family of stabilizer codes in which the qubits are organized on a lattice modeled by the additive group \(\mathbb{Z}^D\) for
+  spatial dimension \(D\), each lattice point contains a finite number \(q\) qubits, and the stabilizer group is translationally
+  invariant with respect to the lattice.
+  
+  A single-qubit Pauli operator can be specified by its lattice coordinate of the site and its \(\mathbb{F}_2^{2q}\) symplectic vector
+  representation of the Pauli operator within the site. The stabilizer group of the translation invariant code is then generated by
+  site-local Pauli operators and their translations. As each lattice coordinate can be represented by a Laurent monomial of \(D\)
+  formal variables, the multiplicative group of finitely supported Pauli operators modulo phase factors on the lattice of dimension
+  \(D\) with \(q\) qubits per site is isomorphic to the additive group of Laurent polynomial column vectors of length \(2q\) in \(D\)
+  formal variables.
+  
+  As an example, \(P(1, 2, 3; e_1)P(-1, 2, 4; e_{q+2})\) is the Pauli operator that applies \(X\) to qubit \(1\) of
+  lattice point \((1, 2, 3)\) and applies \(Z\) to qubit \(2\) of lattice point \((-1, 2, 4)\). This operator can be equivalently
+  expressed by the Laurent polynomial \(x_1x_2^2x_3^3e_1 + x_1^{-1}x_2^2x_3^4e_{q+2}\).
+  
+  For \(D = 2\), translationally invariant stabilizer code Hamiltonians with topological order are equivalent to multiple copies of the
+  toric code\cite{arXiv:1812.11193}.
+  
+  For \(D = 3\), translationally invariant stabilizer code Hamiltonians with topological order can be characterized by four coarse
+  classes\cite{arXiv:1908.08049}:
+  
+  1. Topological quantum field theory: Excitations are mobile in all 3 dimensions. Such codes are conjectured to be equivalent to
+  multiple copies of the 3D toric code.
+  
+  2. Foliated type-I: Excitations are mobile in less than 3 dimensions, but codes can be grown by "foliation"—stacking copies of the 2D
+  toric code.
+  
+  3. Fractal type-I: Excitations are mobile in less than 3 dimensions, and codes are not foliated.
+  
+  4. Type-II: Excitations are not mobile in any dimension.
+
+relations:
```

Comment:
Keep parents/cousins in the original file I just made. Re D=0 case "reducing" to stabilizers, I'm not to keen on having that be a parent distinction; do you feel strongly about this? I propose we put this idea that one can think of stabilizer codes as 0D TI stabs in the description of this class.